torch/_torch_docs.py

```diff
@@ -3594,6 +3594,64 @@ def merge_dicts(*dicts):
     RuntimeError: bool value of Tensor with no values is ambiguous
 """.format(**common_args))
 
+add_docstr(torch.kron,
+           r"""
+kron(input, other, *, out=None) -> Tensor
+
+Computes the Kronecker product, denoted by :math:`\otimes`, of :attr:`input` and :attr:`other`.
+
+If :attr:`input` is a :math:`(a_0 \times a_1 \times \dots \times a_n)` tensor and :attr:`other` is a
+:math:`(b_0 \times b_1 \times \dots \times b_n)` tensor, the result will be a
+:math:`(a_0*b_0 \times a_1*b_1 \times \dots \times a_n*b_n)` tensor with the following entries:
+
+.. math::
+    (\text{input} \otimes \text{other})_{k_0, k_1, \dots, k_n} =
+        \text{input}_{i_0, i_1, \dots, i_n} * \text{other}_{j_0, j_1, \dots, j_n},
+
+where :math:`k_t = i_t * b_t + j_t` for :math:`0 \leq t \leq n`.
+If one tensor has fewer dimensions than the other it is unsqueezed until it has the same number of dimensions.
+
+Supports real-valued and complex-valued inputs.
+
+.. note::
+    The Kronecker product is commonly defined only for matrices.
```

Comment:
Want to suggest a slight tweak here to highlight the relationship between the generalized Kronecker product this implements and the definition for two matrices. Let me know what you think. 

"The Kronecker product is commonly defined only for matrices. If ..." -> "This function generalizes the typical definition of the Kronecker product for two matrices to two tensors, as described above. When :attr:\`input\` is..."

Reply:
Yes, it looks better with your suggestion. I've changed the sentence now.